pyanaconda/ui/gui/spokes/storage.py

```diff
@@ -308,6 +308,34 @@ def execute(self):
         threadMgr.add(AnacondaThread(name=constants.THREAD_EXECUTE_STORAGE,
                                      target=self._doExecute))
 
+        # Register iSCSI to kickstart data
+        iscsi_devices = []
+        # Find all selected disks and add all iscsi disks to iscsi_devices list
+        for d in [d for d in getDisks(self.storage.devicetree) if d.name in self.selected_disks]:
+            # Get parents of a multipath devices
+            if isinstance(d, MultipathDevice):
+                for parent_dev in d.parents:
+                    if isinstance(parent_dev, iScsiDiskDevice) and not parent_dev.ibft:
+                        iscsi_devices.append(parent_dev)
+            # Add no-ibft iScsiDiskDevice. IBFT disks are added by firmware so there is
```

Comment:
To be precise they are not added "by firmware" but "automatically" (by anaconda finding them in iBFT).